src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +37,13 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    /**
+     * Return true if two set of strings are not disjoint, regardless of capitalization.
+     */
+    public static boolean isCaseInsensitiveMatched(Set<String> firstSet, Set<String> secondSet) {
+        List<String> firstSetInLowerCase = firstSet.stream().map(String::toLowerCase).collect(Collectors.toList());
+        List<String> secondSetInLowerCase = secondSet.stream().map(String::toLowerCase).collect(Collectors.toList());
+        return Collections.disjoint(firstSetInLowerCase, secondSetInLowerCase);
```

Comment:
Shouldn't this return true if set of strings are disjoint? 
Collections.disjoint(..) returns true if the two specified collections have no elements in common.

Header comments does not match.

Reply:
Oh yea I didn't notice that!